BinaryObjectScanner/Protection/CrypKey.cs

```diff
@@ -1,13 +1,19 @@
-﻿using BinaryObjectScanner.Interfaces;
+﻿#if NET40_OR_GREATER || NETCOREAPP
+using System.Collections.Concurrent;
+#endif
+using BinaryObjectScanner.Interfaces;
+using SabreTools.Matching;
 using SabreTools.Serialization.Wrappers;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Fix ordering: System, everything else alphabetical. The `#if` goes where it would in the normal set of System